src/apps/backend/modules/account/account_service.py

```diff
@@ -18,6 +18,44 @@ def create_account(*, params: CreateAccountParams) -> Account:
       last_name=account_dict.get("last_name"),
       username=account_dict.get("username"),
     )
+    
+  @staticmethod
+  def get_account_by_username(*, username: str) -> Account:
+    account = AccountReader.get_account_by_username(username=username)
+    account_dict = json.loads(account.to_json())
+    return Account(
+      id=account_dict.get("id"),
+      first_name=account_dict.get("first_name"),
+      last_name=account_dict.get("last_name"),
+      username=account_dict.get("username"),
+    )
+    
+  @staticmethod
+  def reset_account_password(*, params: ResetPasswordParams):
+    from modules.password_reset_token.password_reset_token_service import PasswordResetTokenService
```

Comment:
Hmm, import within a function 🤔

Reply:
If we are encountering a circular dependency, that's a big structural problem really and we should find a way to decople these services.

Reply:
Found a way to remove circular import